config/config.txt

```diff
@@ -22,7 +22,7 @@ $include interviews.txt
 # SERVERSQLNAME tgstation
 
 ## Station name: The name of the station as it is referred to in-game. If commented out, the game will generate a random name instead.
-STATIONNAME Space Station 13
+#STATIONNAME Space Station 13
```

Comment:
why are you editing config files

Reply:
It was back when I asked Wind if he wanted me to do this, and it was so people would receive the changes on their end when they'd download the repo. I'll probably revert that, though.